lms/djangoapps/ccx/plugins.py

```diff
@@ -28,5 +29,7 @@ def is_enabled(cls, course, user=None):
             return True
```

Comment:
@amir-qayyum-khan could you please update the docs to better explain what is happening here? I feel like there are some use cases that are not fully explained and I would like a deeper understanding. I think you could refactor the code here to be clearer as well.

Something else I'm unclear about, why would we return true for this method before we check to see is enabled?